tools/replay.cpp

```diff
@@ -277,9 +277,11 @@ int main(int ac, char** av)
   if ( ac != 2 )
     Replay::usage_and_exit( "Incorrect number of program arguments" );
 
+#if !defined(UMPIRE_ENABLE_ROCM)
```

Comment:
Is this bit necessary right now?

Reply:
Yes, otherwise the replay tests will fail on ARM.

Reply:
Makes sense 👍